c2corg_ui/static/js/constants.js

```diff
@@ -31,6 +31,7 @@ app.constants = {
   REQUIRED_FIELDS: {
     'waypoints': ['title' , 'lang', 'waypoint_type', 'elevation', 'longitude', 'latitude'],
     'articles': ['title', 'activities', 'categories', 'article_type'],
+    'books': ['title', 'activities', 'book_types'],
```

Comment:
"types" indeed https://github.com/c2corg/v6_api/blob/books/c2corg_api/models/book.py#L28
(a book may be of several types)